docs/getting-started-guides/vsphere.md

```diff
@@ -40,201 +40,135 @@ guide](/docs/concepts/storage/volumes/#vspherevolume)
-[Disk]
-    scsicontrollertype = pvscsi
+govc vm.change -e="disk.enableUUID=1" -vm='VM Path'
 ```
+Note: If Kubernetes Node VMs are created from template VM then `disk.EnableUUID=1` can be set on the template VM. VMs cloned from this template, will automatically inherit this property.
 
-* Set the cloud provider via ```--cloud-provider=vsphere``` flag for each instance of kubelet, apiserver and controller manager.
-
-* When upgrading to 1.6 install the default storage class addons, [click here for more details](https://github.com/kubernetes/kubernetes/issues/40070)
-
-#### Known issues
-
-### Kube-up (Deprecated)
-
-Kube-up.sh is no longer supported and is deprecated. The steps for kube-up are included but going forward [kube-anywhere](https://github.com/kubernetes/kubernetes-anywhere) is preferred.
-
-The recommended version for kube-up is [v1.4.7](https://github.com/kubernetes/kubernetes/releases/tag/v1.4.7)
-
-The example below creates a Kubernetes cluster with 4 worker node Virtual.
-Machines and a master Virtual Machine (i.e. 5 VMs in your cluster). This cluster is set up and controlled from your workstation (or wherever you find convenient).
 
-#### Prerequisites
+**[ Step 4 ]** Create and assign Roles to the vSphere Cloud Provider user and vSphere entities.
 
-* You need administrator credentials to an ESXi machine or vCenter instance with write mode api access enabled (not available on the free ESXi license).
-* You must have Go (see [here](https://git.k8s.io/community/contributors/devel/development.md#go-versions) for supported versions) installed: [www.golang.org](http://www.golang.org).
-* You must have your `GOPATH` set up and include `$GOPATH/bin` in your `PATH`.
+Note: if you want to use Administrator account then this step can be skipped.
 
-```shell
-export GOPATH=$HOME/src/go
-mkdir -p $GOPATH
-export PATH=$PATH:$GOPATH/bin
-```
-
-* Install the govc tool to interact with ESXi/vCenter. Head to [govc Releases](https://github.com/vmware/govmomi/releases) to download the latest.
-
-```shell
-# Sample commands for v0.8.0 for 64 bit Linux.
-curl -OL https://github.com/vmware/govmomi/releases/download/v0.8.0/govc_linux_amd64.gz
-gzip -d govc_linux_amd64.gz
-chmod +x govc_linux_amd64
-mv govc_linux_amd64 /usr/local/bin/govc
-```
-
-* Get or build a [binary release](/docs/getting-started-guides/binary_release)
+vSphere Cloud Provider requires the following minimal set of privileges to interact with vCenter. Please refer [vSphere Documentation Center](http://pubs.vmware.com/vsphere-65/index.jsp?topic=%2Fcom.vmware.vsphere.security.doc%2FGUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html&resultof=%22%43%72%65%61%74%65%22%20%22%63%72%65%61%74%22%20%22%43%75%73%74%6f%6d%22%20%22%63%75%73%74%6f%6d%22%20%22%52%6f%6c%65%22%20%22%72%6f%6c%65%22%20) to know about steps for creating a Custom Role, User and Role Assignment.
```

Comment:
Nit: this URL is ugly.  It seems to redirect to a much prettier: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.security.doc/GUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html